### ISSUE TITLE
jquery-tablesort sorts incorrectly

### ISSUE THREAD

**Issue by samuelcolvin** [kylefox/jquery-tablesort#31]
Just started using jquery-tablesort but sadly it sorts incorrectly for the following table. The issue seems to be the use of `<th scope="row">1</th>` on rows. (html taken directly from the bootstrap 3 docs, so it should be well formed.)

```html
  <table class="table">
    <thead>
      <tr>
        <th>#</th>
        <th>First Name</th>
        <th>Last Name</th>
        <th>Username</th>
      </tr>
    </thead>
    <tbody>
    <tr>
      <th scope="row">1</th>
      <td>Mark</td>
      <td>Otto</td>
      <td>@mdo</td>
    </tr>
    <tr>
      <th scope="row">2</th>
      <td>Jacob</td>
      <td>Thornton</td>
      <td>@fat</td>
    </tr>
    <tr>
      <th scope="row">3</th>
      <td>Larry</td>
      <td>the Bird</td>
      <td>@twitter</td>
    </tr>
    </tbody>
  </table>

**Comment by kylefox** [comment]
I just did a quick [CodePen](http://codepen.io/anon/pen/ZOkwBX) and it looks like you're correct.

Unfortunately I don't have time to debug/patch this myself — pull requests welcome.

**Comment by kylefox** [comment]
Fixed in [v0.0.11](https://github.com/kylefox/jquery-tablesort/tree/v0.0.11).
